### PR TITLE
Adding Docker Plugin To Build.sbt

### PR DESCRIPTION
**Pull request by shankarshastri**
This PR is all about using sbt-native-packager for constructing docker image as part of the g8 template.

### CODE REVIEW

**Comment by lightbend-cla-validator** [comment]
Hi @shankarshastri,

Thank you for your contribution! We really value the time you've taken to put this together.

Before we proceed with reviewing this pull request, please sign the Lightbend Contributors License Agreement:

[http://www.lightbend.com/contribute/cla](http://www.lightbend.com/contribute/cla)

File: src/main/g8/src/main/scala/$package$/QuickstartServer.scala [hunk 1]
```diff
@@ -20,16 +21,17 @@ object QuickstartServer extends App with UserRoutes {
...
   // from the UserRoutes trait
   lazy val routes: Route = userRoutes
   //#main-class
 
   //#http-server
-  Http().bindAndHandle(routes, "localhost", 8080)
+  Http().bindAndHandle(routes, bindingAddress, 8080)
 
-  println(s"Server online at http://localhost:8080/")
+  println(s"Server online at http://\$bindingAddress:8080/")
```
Comment (raboof) [inline src/main/g8/src/main/scala/$package$/QuickstartServer.scala]: This will not show up correctly in the paradox-generated documentation unfortunately.

Comment (shankarshastri) [reply src/main/g8/src/main/scala/$package$/QuickstartServer.scala]: Correct. I agree with you. I may require your inputs on how to solve this one.

**Comment by shankarshastri** [comment]
@ktoso, your inputs?

**Comment by patriknw** [comment]
I agree, can be part of some other guide but not quickstart (the very first experience for new users)

**Pull request closed without merging**